gems/sorbet/lib/gem-generator-tracepoint/tracer.rb

```diff
@@ -14,6 +14,12 @@ def DelegateClass(superclass)
   end
 end
 
+# Only define the `ruby2_keywords` method if the Ruby version is less than 2.7
+# and the method isn't already defined.
+if RUBY_VERSION < '2.7' && !respond_to?(:ruby2_keywords, true)
+  def ruby2_keywords(*); end
+end
+
```

Comment:
I don't think this is a very good thing to do. We already have places in `sorber-runtime` that use `ruby2_keywords` and every usage works by checking `respond_to?`.